spec/test.spec.js

```diff
@@ -190,22 +190,23 @@ describe('S3Adapter tests', () => {
     });
 
     it('should accept endpoint as an override option in args', () => {
```

Comment:
Looks like the `AWS.Endpoint` API simply disappeared and I am not sure what to do here.